hc/accounts/views.py

```diff
@@ -187,6 +188,19 @@ def profile(request):
                                       user=farewell_user).delete()
 
                 messages.info(request, "%s removed from team!" % email)
+        elif "select_allowed_checks" in request.POST:
+            post_data = dict(request.POST.lists())
+            check_ids = post_data.get('checks', None)
+            member_id = post_data.get('member_id', None)
+            member = User.objects.get(id=member_id[0])
+            # remove previous entries
+            for allowed_check in MemberAllowedChecks.objects.filter(user=member):
```

Comment:
We don't require iterating over the queryset to delete
https://docs.djangoproject.com/en/dev/topics/db/queries/#deleting-objects